themes/bootstrap3/js/lightbox.js

```diff
@@ -318,9 +318,9 @@ VuFind.register('lightbox', function Lightbox() {
       }, false);
     }
     // Loading
-    var lightboxHeader = $(_modalBody).find('.lightbox-header');
-    if (lightboxHeader.length > 0) {
-      lightboxHeader.after(VuFind.loading());
+    $(_modalBody).find('.modal-loading-overlay,.loading-spinner').remove();
```

Comment:
Just a small notice, but the conversion of _modalBody to jQuery object might be not necessary?

Reply:
Thanks, fixed!